app/views/ui-components/v1/cards/_metal_level_select.html.slim

```diff
@@ -186,6 +186,9 @@ javascript:
   var sponsorContributions = #{@benefit_package_form.catalog.sponsor_contributions(@benefit_package_form.id).to_json.html_safe};
   var minContributions = #{f.object.sponsor_contribution.min_contributions_map.to_json.html_safe};
 	var productsTotal;
+  const isPvpFeatureEnabled = "#{::EnrollRegistry.feature_enabled?(:premium_value_products)}";
```

Comment:
I'd omit the quotes and have this be an actual boolean, to avoid confusion and a more complex type. The `isxxx` naming in particular suggests a boolean (like with `plan.is_pvp_eligible` below`) to me.